### PR TITLE
Fix Export SIF

### PR DESCRIPTION
**Pull request by ntran18**
Previous Pull Request only fix the export when importing the sif file and export the same sif file. But if you pull data from database, and export it, the export doesn't say 'pp'. 

This new commit allows export SIF -> SIF and Excel workbook -> SIF, also adding constant variables for GRN and PPI

### CODE REVIEW

**Comment by ntran18** [comment]
Pull Request Checklist
- [x] Travis C.I. build passes
- [x] All five Demos look as expected when loaded
- [x] Reload works as expected
- [x] Loading file with Error brings up error modal, and it looks as expected
- [x] Loading file with Warning brings up warnings modal
- [x] Loading a network from the database looks as expected when loaded
- [x] Import works for both SIF and GraphML
- [x] Graph can be exported to SIF, GraphML and Excel
- [x] Graph can be exported to PNG, SVG and PDF
- [x] Expression data and Additional Sheets can be exported to Excel
- [x] Print works as expected
- [x] Restrict graph to viewport works as expected (check/uncheck)
- [x] Viewport Size Changing works as expected (small/medium/large/fit)
- [x] Toggle between Grid Layout and Force Graph Layout works as expected
- [x] Force Graph Parameter Sliders change the number above them and have an effect on the graph
- [x] Locking/Unlocking/Resetting/Undo Resetting the force graph parameters works as expected
- [x] Enabling and disabling node coloring works as expected
- [x] Node coloring options work as expected (selection top/bottom dataset, averaging values, changing max value)
- [x] Weighted graph loaded with the "Enable Edge Coloring" option checked appears as expected
- [x] Hide/Show Weights works as expected (always/never/upon mouseover)
- [x] Edge Weight Normalization Factor can be changed (set/reset)
- [x] Gray Edge Threshold can be changed, slider changes the number and has an effect on the graph
- [x] Checking Show Gray Edges as Dashed works as expected (check/uncheck)
- [x] D-pad left/right/top/bottom/center works as expected
- [x] Zoom slider changes number and has effect on graph (viewport and menu)
- [x] Right click on a node opens gene page, page is populated with correct data

File: server/controllers/exporters/xlsx.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 // const { meta } = require("eslint/lib/rules/*");
 const xlsx = require("node-xlsx");
+<<<<<<< HEAD
+const { CELL_A1_GRN, CELL_A1_PPI, NETWORK_GRN_MODE } = require("../constants");
+=======
 const { CELL_A1_GRN, CELL_A1_PPI } = require("../constants");
+>>>>>>> beta
```
Comment (dondi) [inline server/controllers/exporters/xlsx.js:7]: Oops watch out! Looks like an unresolved merge conflict here

File: server/controllers/network-sheet-parser.js [hunk 2]
```diff
@@ -2,7 +2,11 @@
 // var path = require("path");
 // var demoWorkbooks = require(__dirname + "/demo-workbooks");
 
+<<<<<<< HEAD
+const { NETWORK_PPI_MODE, NETWORK_GRN_MODE, CELL_A1_GRN, CELL_A1_PPI } = require("./constants");
+=======
 const { CELL_A1_GRN, CELL_A1_PPI } = require("./constants");
+>>>>>>> beta
```
Comment (dondi) [inline server/controllers/network-sheet-parser.js:9]: Another one…

File: server/controllers/network-sheet-parser.js [hunk 3]
```diff
@@ -306,9 +310,15 @@ exports.workbookType = function (workbookFile) {
...
             if (cellA1 === CELL_A1_GRN) {
+<<<<<<< HEAD
+                workbookType = NETWORK_GRN_MODE;
+            } else if (cellA1 === CELL_A1_PPI) {
+                workbookType = NETWORK_PPI_MODE;
+=======
                 workbookType = "grn";
             } else if (cellA1 === CELL_A1_PPI) {
                 workbookType = "protein-protein-physical-interaction";
+>>>>>>> beta
```
Comment (dondi) [inline server/controllers/network-sheet-parser.js:321]: Hmmm…either this is a missed commit or a missed test-run before pushing 😅

**Review by dondi — COMMENTED** [review]
Generally looks good except for the missed merge conflict markers 😬—please give those a run and make sure to execute things locally to ensure I didn’t miss others too…

File: server/controllers/exporters/xlsx.js (hunk 1 above)
Comment (ntran18) [reply server/controllers/exporters/xlsx.js:7]: Fixed

File: server/controllers/network-sheet-parser.js (hunk 3 above)
Comment (ntran18) [reply server/controllers/network-sheet-parser.js:321]: I don't really know what issue. It passed all the tests and lint, and pass merge conflict too. Fixed

File: server/controllers/network-sheet-parser.js (hunk 2 above)
Comment (ntran18) [reply server/controllers/network-sheet-parser.js:9]: Fix...

**Review by dondi — APPROVED** [review]
LGTM

**Pull request merged**